rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly-2021-04-18
```

Comment:
Why is this necessary

Reply:
It isn't strictly speaking but it does tell people what a known-working compiler version is. I consider this more important when something is nightly-only. YMMV.

You can remove it before merge or I can. Whatever you want.